public/components/report_definitions/report_trigger/report_trigger.tsx

```diff
@@ -76,10 +101,25 @@ export function ReportTrigger() {
 
   const handleReportTriggerType = (e: React.SetStateAction<string>) => {
     setReportTriggerTypes(e);
+    if (e === 'scheduleOption') {
+      trigger_schema['trigger_type'] = 'Schedule';
+    } else if (e === 'alertOption') {
+      trigger_schema['trigger_type'] = 'Alert';
+    }
   };
 
   const handleScheduleRequestTime = (e: React.SetStateAction<string>) => {
     setScheduleRequestTime(e);
+    if (e === 'nowOption') {
+      trigger_params['schedule_type'] = 'Now';
+      createReportDefinitionRequest['report_type'] = 'Download';
+    } else if (e === 'futureDateOption') {
+      trigger_params['schedule_type'] = 'Future date';
+    } else if (e === 'recurringOption') {
+      trigger_params['schedule_type'] = 'Recurring';
+    } else if (e === 'cronBasedOption') {
+      trigger_params['schedule_type'] = 'Cron';
+    }
```

Comment:
use a map for readability, refer to below
```
const optionMap = {
  "nowOption": "now",
  "futureDateOption": "Future date",
   ...
}

trigger_params['schedule_type'] = optionMap[e];
if (e === 'nowOption') {
  reateReportDefinitionRequest['report_type'] = 'Download';
}

```

Reply:
Good suggestion- will change